src/pages/tutorials/smart-rollups/debug.md

```diff
@@ -0,0 +1,38 @@
+---
+id: debug
+title: "Part 2: Running the kernel in debug mode"
+lastUpdated: 25th October 2023
+---
+
+Octez provides a command named `octez-smart-rollup-wasm-debugger` that runs smart rollups in debug mode to make it easier to test and observe them.
+Later, you will deploy the rollup to the sandbox, but running it in debug mode first verifies that it built correctly.
+
+1. In the terminal window inside the Docker container, go to the `hello_world_kernel` folder.
+
+1. Run this command to start the rollup and pass an empty message inbox to it:
+
+   ```bash
+   octez-smart-rollup-wasm-debugger \
+     --kernel target/wasm32-unknown-unknown/debug/hello_world_kernel.wasm \
+     --inputs empty_input.json
+   ```
```

Comment:
an error appeared when I try this command:
/bin/sh: step: not found
~/hello-world-kernel $ octez-smart-rollup-wasm-debugger \
>   --kernel target/wasm32-unknown-unknown/debug/hello_world_kernel.wasm \
>   --inputs empty_input.json
Error
  Wrong value for command line option --kernel.
  target/wasm32-unknown-unknown/debug/hello_world_kernel.wasm is not a valid file

Reply:
I get that error when the .wasm file is not found. Are you sure you are in the hello_world_kernel folder and the path to the wasm file is correct?